bloomfilter/src/Data/BloomFilter/Blocked/Calc.hs

```diff
@@ -0,0 +1,98 @@
+-- | Various formulas for working with bloomfilters.
+module Data.BloomFilter.Blocked.Calc (
+    NumEntries,
+    BloomSize (..),
+    FPR,
+    sizeForFPR,
+    BitsPerEntry,
+    sizeForBits,
+    sizeForPolicy,
+    BloomPolicy (..),
+    policyFPR,
+    policyForFPR,
+    policyForBits,
+) where
+
+import           Data.BloomFilter.Classic.Calc (BitsPerEntry, BloomPolicy (..),
+                     BloomSize (..), FPR, NumEntries)
+
+policyForFPR :: FPR -> BloomPolicy
+policyForFPR fpr | fpr <= 0 || fpr >= 1 =
+    error "bloomPolicyForFPR: fpr out of range (0,1)"
+
+policyForFPR fpr =
+    BloomPolicy {
+      policyBits   = c,
+      policyHashes = k
+    }
+  where
+    k       :: Int
+    k       = max 1 (round (recip_log2 * log_fpr))
+    c       = log_fpr * log_fpr * f2
+            +           log_fpr * f1
+            +                     f0
+    log_fpr = negate (log fpr)
+
+    -- These parameters are from a (quadratic) linear regression in log space
+    -- of samples of the actual FPR between 1 and 20 bits. This is with log FPR
+    -- as the indepedent variable and bits as the depedent variable.
+    f2,f1,f0 :: Double
+    f2 = 8.035531421107756e-2
+    f1 = 1.653017726702572
+    f0 = 0.5343568065075601
+{-
+Regression, FPR indepedent, bits depedent:
+Fit {fitParams = V3 8.035531421107756e-2 1.653017726702572 0.5343568065075601, fitErrors = V3 7.602655075308541e-4 8.422591688796256e-3 2.0396917012822195e-2, fitNDF = 996, fitWSSR = 18.362899348627252}
+Fit {fitParams = V3 8.079418894776325e-2 1.6462569292513933 0.5550062950289885, fitErrors = V3 7.713375250014809e-4 8.542261871094414e-3 2.0678969159415226e-2, fitNDF = 996, fitWSSR = 19.00125036371992}
+-}
```

Comment:
Were these left in intentionally or accidentally?

Reply:
Yes :-)

They're the output of two runs of the fpr-calc program to generate a regression fit. It's not strictly necessary that they be recorded here, but alternatively it might be nice to add a proper comment about how the magic numbers were obtained (including saved output) so people can reproduce it or change it in future.